modules/test/playwright/tests/object-web/env/portal-ext.properties

```diff
@@ -0,0 +1,2 @@
+object.encryption.algorithm=AES
+object.encryption.key=D9z5Rwxkn+8SctNWW/q/OA==
```

Comment:
Hey @igor-franca, Just one question, is there any problem with exposing this encryption key in a public repository? Is this sensitive information?

Reply:
Hey Dan, I got these values ​​from Liferey Learn, they are public information.